addons/auth_passkey/security/security.xml

```diff
@@ -3,7 +3,24 @@
     <record model="ir.rule" id="rule_auth_passkey_key_user">
         <field name="name">Passkeys: Users can only access own Passkeys</field>
         <field name="model_id" ref="model_auth_passkey_key"/>
-        <field name="groups" eval="[(4, ref('base.group_user'))]"/>
+        <field name="groups" eval="[
+            (4, ref('base.group_portal')),
+            (4, ref('base.group_user')),
+        ]"/>
+        <field name="domain_force">[('create_uid', '=', user.id)]</field>
+        <field name="perm_unlink" eval="1"/>
+        <field name="perm_write" eval="1"/>
+        <field name="perm_read" eval="1"/>
+        <field name="perm_create" eval="1"/>
```

Comment:
No need to precise `<field name="perm_create" eval="1"/>` this is the default value

I believe you will need a migration script to force the update of `auth_passkey.rule_auth_passkey_key_user` as it's `noupdate="1"`